### PR TITLE
feat: disable time cache to improve performance

### PR DESCRIPTION
**Pull request by ansiz**
### Describe what this PR does / why we need it

#441 

### Special notes for reviews

### CODE REVIEW

**Review by louyuting — APPROVED** [review]
LGTM

File: core/config/entity.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func NewDefaultConfig() *Entity {
 					CollectMemoryIntervalMs: DefaultMemoryStatCollectIntervalMs,
 				},
 			},
-			UseCacheTime: true,
+			UseCacheTime: false,
```
Comment (sanxun0325) [inline core/config/entity.go:143]: If not set, the default value is false

**Review by sanxun0325 — APPROVED** [review]

File: core/config/entity.go (hunk 1 above)
Comment (ansiz) [reply core/config/entity.go:143]: 显式的初始化似乎没有啥坏处？

**Review by luckyxiaoqiang — APPROVED** [review]
LGTM

**Review by sczyh30 — APPROVED** [review]
LGTM

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**